js/pro/coinex.js

```diff
@@ -499,8 +526,60 @@ module.exports = class coinex extends coinexRest {
                 this.safeInteger (timeframes, timeframe, timeframe),
             ],
         };
+        const subscription = {
+            'symbol': symbol,
+            'timeframe': timeframe,
+        };
         const request = this.deepExtend (subscribe, params);
-        const ohlcvs = await this.watch (url, messageHash, request, messageHash);
+        const ohlcvs = await this.watch (url, messageHash, request, messageHash, subscription);
+        if (this.newUpdates) {
+            limit = ohlcvs.getLimit (symbol, limit);
+        }
+        return this.filterBySinceLimit (ohlcvs, since, limit, 0, true);
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
```

Comment:
Is there already `fetchOHLCV` in coinex exchange?

Reply:
Yes, though this is to query through the websocket channel. After the conversation in the team channel we decided to add websocket queries in this manner.